src/transformers/models/gpt_neo/modeling_gpt_neo.py

```diff
@@ -491,14 +477,23 @@ def _upad_input(self, query_layer, key_layer, value_layer, attention_mask, query
 
 
 class GPTNeoAttention(nn.Module):
-    def __init__(self, config, layer_id=0):
```

Comment:
uhmmm... I think we can leave the old `layer_id` arg as it was, as it is less BC breaking!

We wouldn't need the warning in this case.

Reply:
oke done